edx_reverification_block/xblock/static/js/skip_reverification.js

```diff
@@ -1,25 +1,35 @@
-function SkipReverifcation(runtime, element) {
-  $(element).find('#skip').bind('click', function() {
-      $(element).find('#skip-reverification-modal').show();
+function SkipReverification(runtime, element) {
+  var $el = $(element);
+
+  var setSkipConfirmVisible = function(isVisible) {
+    var skipConfirm = $el.find('.reverify-skip-confirm-wrapper'),
+        reverifyMsg = $el.find('.reverify-now-wrapper');
+    if (isVisible) {
+      reverifyMsg.addClass('hidden');
```

Comment:
Any reason to not use jQuery [.hide()](http://api.jquery.com/hide/) and [.show()](http://api.jquery.com/show/)?